firebase-sessions/api.txt

```diff
@@ -17,3 +17,51 @@ package com.google.firebase.sessions {
 
 }
 
+package com.google.firebase.sessions.api {
```

Comment:
This was auto-generated by `./gradlew :firebase-sessions:generateApiTxtFile` but it seems fine if I delete all these internal apis.

Reply:
What are the legit APIs that should find a place here? Secondly, how do we control what does inside this file? We can do this as a separate validation task before we go out for launch.

Reply:
Ideally I would like this to not expose any public API since it's for internal products only. But we need at least this to be public for Firebase to work with it.